cmd/harbor/root/config/update.go

```diff
@@ -0,0 +1,267 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package config
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/goharbor/harbor-cli/pkg/utils"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// SetConfigItemCommand creates the 'harbor config set' subcommand,
+// allowing you to do: harbor config set <item> <value>.
+func SetConfigItemCommand() *cobra.Command {
+	var credentialName string
+
+	cmd := &cobra.Command{
+		Use:   "set <item> <value>",
```

Comment:
Instead of set we should call it update here.

```suggestion
		Use:   "update <item> <value>",
```